packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/components/StepSection.tsx

```diff
@@ -16,13 +18,22 @@ const StepSection: React.FunctionComponent<StepSectionProps> = ({
   isOpen,
   onOpenStep,
   children,
+  status = null,
 }) => {
   const stepSectionContainer = useRef<HTMLDivElement>(null);
   const contentHeight: number =
     !!stepSectionContainer && !!stepSectionContainer.current
       ? stepSectionContainer.current.scrollHeight
       : 0;
 
+  const additionalStepTitleStyle: ThemeUIStyleObject =
+    status === "ok"
```

Comment:
The status values are typed with a union "ok" | "ko" | null, try to put something else and you will have a typescript error on the compilation. 

I choose a union instead of a enum because : 
- that avoid importing the enum everytime we want to make a assertion
- each step status value remain independent, if one day we want to have a check with a new status or less status we can do it easily without creating sub enum or specific enum

To sum-up with this approach we ensure we will always use the right value (type checked by typescript) without adding any complexity and that add a little bit of flexibility for later on if we need it.